### PR TITLE
Removing Data Science Pipelines' manifests

### PR DESCRIPTION
**Pull request by DharmitD**
Removing Data Science Pipelines' manifests from odh-manifests repository, the manifests will be managed by [the new Data Science Pipelines Operator](https://github.com/opendatahub-io/data-science-pipelines-operator).
Keeping the Data Science Pipelines' standalone (operator-less) manifests in the Data Science Pipelines' ([upstream](https://github.com/opendatahub-io/data-science-pipelines-operator) & [downstream](https://github.com/red-hat-data-services/data-science-pipelines-operator)) repositories.

Closes [RHODS-6927](https://issues.redhat.com/browse/RHODS-6927)

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message
- [x] JIRA link(s): [RHODS-6927](https://issues.redhat.com/browse/RHODS-6927)
- [ ] The Jira story is acked
- [ ] An entry has been added to the latest build document in [Build Announcements Folder](https://drive.google.com/drive/folders/1sgkK1WZgGo9CXsLizNe0GbAzVKuSKrZL).
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious)

### CODE REVIEW

**Comment by HumairAK** [comment]
Lgtm,

blocked by: 

https://github.com/opendatahub-io/odh-manifests/pull/737
https://github.com/opendatahub-io/odh-manifests/pull/729
and once 729 is added to downstream

**Comment by LaVLaS** [comment]
This can merge downstream since upstream needs to gracefully handle the deprecation of this manifest.

**Comment by DharmitD** [comment]
@LaVLaS @anishasthana @VaishnaviHire this PR is now ready to be merged, thanks.

File: tests/setup/kfctl_openshift.yaml [hunk 1]
```diff
@@ -47,17 +47,5 @@ spec:
...
-        - metadata-store-mariadb
-        - ds-pipeline-ui
-        - object-store-minio
-        - default-configs
-      repoRef:
-        name: manifests
-        path: data-science-pipelines
-    name: data-science-pipelines
-  repos:
-  - name: manifests
```
Comment (HumairAK) [inline tests/setup/kfctl_openshift.yaml:61]: @DharmitD you're removing lines that are not part of the `kustomizeConfig` 

```
  repos:
  - name: manifests
```

Comment (DharmitD) [reply tests/setup/kfctl_openshift.yaml]: Done, added these back.

**Review by HumairAK — APPROVED** [review]
/lgtm

**Pull request merged**